Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -179,63 +179,105 @@ public class PublishPacket: CStruct {
+                                                                    payloadFormatIndicator?.nativeValue ?? nil
                 let _messageExpiryInterval: UInt32? = try? messageExpiryInterval?.secondUInt32() ?? nil
 
                 return withOptionalUnsafePointers(
                     _payloadFormatIndicatorInt,
                     topicAlias,
                     _messageExpiryInterval) { payloadPointer, topicAliasPointer, messageExpiryIntervalPointer in
-                    if let _payloadPointer = payloadPointer {
-                        raw_publish_view.payload_format = _payloadPointer
-                    }
-
-                    if let _messageExpiryIntervalPointer = messageExpiryIntervalPointer {
-                        raw_publish_view.message_expiry_interval_seconds = _messageExpiryIntervalPointer
-                    }
-
-                    if let _topicAliasPointer = topicAliasPointer {
-                        raw_publish_view.topic_alias = _topicAliasPointer
-                    }
-
+                    raw_publish_view.payload_format = payloadPointer
+                    raw_publish_view.message_expiry_interval_seconds = messageExpiryIntervalPointer
+                    raw_publish_view.topic_alias = topicAliasPointer
+                        
                     return withOptionalArrayRawPointer(of: subscriptionIdentifiers) { subscriptionPointer in
 
-                        if let _subscriptionPointer = subscriptionPointer {
-                            raw_publish_view.subscription_identifiers = _subscriptionPointer
+                        if let subscriptionPointer = subscriptionPointer {
+                            raw_publish_view.subscription_identifiers = subscriptionPointer
                             raw_publish_view.subscription_identifier_count = subscriptionIdentifiers!.count
                         }
 
                         return withOptionalUserPropertyArray(
                             of: userProperties) { userPropertyPointer in
-                            if let _userPropertyPointer = userPropertyPointer {
+                            if let userPropertyPointer = userPropertyPointer {
                                 raw_publish_view.user_property_count = userProperties!.count
                                 raw_publish_view.user_properties =
-                                    UnsafePointer<aws_mqtt5_user_property>(_userPropertyPointer)
+                                    UnsafePointer<aws_mqtt5_user_property>(userPropertyPointer)
                             }
-                            return withOptionalByteCursorPointerFromString(
+                            return withOptionalByteCursorPointerFromStrings(
                             responseTopic,
-                            correlationData,
-                            contentType) { cResponseTopic, cCorrelationData, cContentType in
+                            contentType) { cResponseTopic, cContentType in
```

Comment:
nit weird format
responseTopic should not align with return and should have at least one tab.